packages/ui/src/ui/pages/query-tracker/QueryEditor/QueryEditor.tsx

```diff
@@ -35,12 +38,60 @@ const QueryEditorView = React.memo(function QueryEditorView({
     const activeQuery = useSelector(getQuery);
     const text = useSelector(getQueryText);
     const engine = useSelector(getQueryEngine);
+    const editorErrors = useSelector(getQueryEditorErrors);
+    const decorationsCollection = useRef<monaco.editor.IEditorDecorationsCollection | undefined>(
+        undefined,
+    );
+
+    const updateErrorMarkers = () => {
+        const model = editorRef.current?.getModel();
```

Comment:
It is better to add model to list of dependencies in line 93.
Also, as I know, `eslint` should highlight some similar problems for useEffect.

Reply:
I've wrapped `updateErrorMarkers` with `useCallback`, and added it as the dependency in line 93👍🏻

Reply:
It is a good practice to add all closured variables to dependencies, at the moment model/editorRef is not added. Does `eslint` hightlight some warning about it?
What is the reason to have `updateErrorMarkers` as a named variable (the function might be used as first argument of useEffect in line 93)?